### PR TITLE
follow button started working

### LINKED ISSUE #4443: follow icon is not working in wiki section
### Please describe the problem (or idea)

> What happened just before the problem occurred? Or what problem could this idea solve?
image

Follow icon is not working/responding 

### Please show us where to look

https://publiclab.org/wiki/comment-templates

### What's your PublicLab.org username?

geekychaser

> This can help us diagnose the issue: 

### Browser, version, and operating system

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site. 

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by geekychaser**
Fixes #4443 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] PR is descriptively titled 📑
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by IshaGupta18** [comment]
@geekychaser have you changes the class of follow button ie. changed the icon from user-plus to eye, to get it working?

**Comment by IshaGupta18** [comment]
@geekychaser you can keep the icon class as it is and just remove the rel="toottip" title="follow" from the original code, because rel="tooltip" was overriding rel="popover", that's why this button wasn't working. If you want to keep the tooltip as well, then you can change rel="popover" to data-toggle="popover".

**Comment by oorjitchowdhary** [comment]
Yes please change it to fa fa-user plus..
Refer #4278

**Comment by geekychaser** [comment]
@oorjitchowdhary yeah okay got it

**Comment by geekychaser** [comment]
image

i guess this looks fine now ?

**Comment by gauravano** [comment]
Hi @geekychaser fa-user looks fine. Reviewing the PR.

File: app/views/like/_like.html.erb [hunk 1]
```diff
@@ -10,9 +10,9 @@
...
-      <i class="fa fa-user-plus" aria-hidden="true"></i> 
-    </li>
+
+    <li class="btn btn-default btn-sm" data-html="true" rel="popover" data-content="<%= "No tags" if tagnames.nil? || tagnames.length == 0 %><% if tagnames %><% tagnames.each do |tagname| %><p style='margin-bottom:3px;'><a href='/subscribe/tag/<%= tagname %>' class='btn btn-default btn-mini'><%= tagname %></a></p><% end %><% end %><hr /><a class='btn btn-sm'><i class='fa fa-user'></i> <%= node.author.name %></a>" title="Follow by tag or author" data-placement="left">
+    <i class="fa fa-user-plus"></i></li>
```
Comment (gauravano) [inline app/views/like/_like.html.erb]:
```suggestion
        <i class="fa fa-user-plus"></i>
    </li>
```

File: app/views/like/_like.html.erb [hunk 2]
```diff
@@ -10,9 +10,9 @@
     <a rel="tooltip" title="Flag as spam" class="btn btn-sm btn-default btn-flag-spam-<%= node.id %>" href="mailto:moderators@publiclab.org?subject=Reporting+spam+on+Public+Lab&body=Hi,+I+found+this+item+that+looks+like+spam+or+needs+to+be+moderated:+<%= node.title.gsub(/ /,'+') %>+https://publiclab.org/n/<%= node.id %>+by+https://publiclab.org/profile/<%= node.author.username %>+Thanks!">
       <i class="fa fa-flag"></i>
     </a>
-    <li rel="tooltip" title="Follow" class="btn btn-default btn-sm" data-html="true" rel="popover" data-content="<%= "No tags" if tagnames.nil? || tagnames.length == 0 %><% if tagnames %><% tagnames.each do |tagname| %><p style='margin-bottom:3px;'><a href='/subscribe/tag/<%= tagname %>' class='btn btn-default btn-mini'><%= tagname %></a></p><% end %><% end %><hr /><i class='fa fa-user'></i><% if current_user && !current_user.following?(node.author.user) %>&nbsp;<a class='btn btn-sm' href='/relationships?followed_id=<%= node.author.id %>' data-method='post' > <%= node.author.name %></a><% else %><%= node.author.name %><% end %>" title="Follow by tag or author" data-placement="left">
```
Comment (gauravano) [inline app/views/like/_like.html.erb]: You have also removed the link behind working of `user` link so please add it again. Thanks!

**Comment by SidharthBansal** [comment]
There are conflicts please remove them.

**Comment by geekychaser** [comment]
@jywarren @gauravano please review.
thank you

**Comment by gauravano** [comment]
Merged :tada: :balloon: !

**Pull request merged**